cmake/CircuitCompile.cmake

```diff
@@ -107,8 +107,8 @@ function(add_circuit)
         set(libcpp_stdlib ${CMAKE_BINARY_DIR}/libs/stdlib/libcpp/zkllvm-libcpp.ll)
     else()
         set(LINKER llvm-link-zkllvm)
-        set(libc_stdlib "/usr/lib/zkllvm/lib/zkllvm/zkllvm-libc.ll")
-        set(libcpp_stdlib "/usr/lib/zkllvm/lib/zkllvm/zkllvm-libcpp.ll")
+        set(libc_stdlib "usr/lib/zkllvm/lib/zkllvm/zkllvm-libc.ll")
+        set(libcpp_stdlib "usr/lib/zkllvm/lib/zkllvm/zkllvm-libcpp.ll")
```

Comment:
Please revert these changes, it requires more accurate fixes.

Reply:
Thanks for suggestion, I have applied it here
https://github.com/NilFoundation/zkLLVM/commit/deba633744f206a50351169e3f42d687a5cf2068